src/main/webapp/community.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Community Page</title>
+    <link rel="stylesheet" href="/css/main.css">
+
+    <script>
+
+      /** Fetches users and adds them to the page. */
+      function fetchUserList(){
+        const url = '/user-list';
+        fetch(url).then((response) => {
+          return response.json();
+        }).then((users) => {
+          const list = document.getElementById('list');
+          list.innerHTML = '';
+
+          users.forEach((user) => {
+           const userListItem = buildUserListItem(user);
+           list.appendChild(userListItem);
+          });
+        });
+      }
+
+      /**
+       * Builds a list element that contains a link to a user page, e.g.
+       * <li><a href="/user-page.html?user=test@example.com">test@example.com</a></li>
```

Comment:
This is a great example of the kind of stuff that is helpful in function comments! Shows me exactly what this function is going to give me without me needing to read any of the details of the function to understand how it works.